frameworklauncher/src/main/java/com/microsoft/frameworklauncher/applicationmaster/Node.java

```diff
@@ -48,7 +48,9 @@ public static Node fromNodeReport(NodeReport nodeReport) throws Exception {
         ResourceDescriptor.fromResource(nodeReport.getUsed()));
   }
 
-  // Compare two node's AvailableResource,  order is Gpu, Cpu, Memory
+  // Compare two node gpu resource.
+  // First compare two nodes available Gpu resource.
+  // Then compare two nodes total Gpu resource,
   @Override
```

Comment:
Keep original comments?
  // Compare two node's AvailableResource,  order is Gpu, Cpu, Memory